packages/admin-ui-docs/docs/templates/listing page/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
The file names are `listing page` can you rename them to `listing-page` instead? Just to follow the pattern that we are using of representing the space with `-`